docs/participate/translate_space_acres.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Space Acres UI Translation Guide
+sidebar_position: 4
+description: Translation Guide for the Space Acres UI
+keywords:
+    - Space Acres UI translation
+    - Community
+    - Autonomys
+    - Translate
+    - Localization
+---
+
+
+## Translation Guide
+
+Thank you for considering contributing to the Space Acres UI locallization! This guide is here to help you contribute to our goal of making farming on the Autonomys Network more accessible and inclusive by localizing or translating the user interface. The Autonomys Network is driven by a vision of a decentralized and equitable future, and we believe that overcoming language barriers is crucial to achieving this vision.
+
+## Before you start
+
+Space Acres has a lot of techncical and Autonomys-related slang that **will not be translated correctly** via the AI-chats similar to **Chat GPT** or **Google Translate**. We kindly ask you to work to do all translations manually, since we can't really verify their correctness. We thank you for your understanding! 
```

Comment:
```suggestion
Space Acres has a lot of technical and Autonomys-related slang that **will not be translated correctly** via LLMs such as **Chat GPT** or translation services such as **Google Translate**. We kindly ask you to work to do all translations manually, since we can't really verify their correctness. We thank you for your understanding! 
```

Reply:
Should we be explicitly stating that we are unable to verify translations? I feel that it is counter-intuitive.